lib/hard_data.dart

```diff
@@ -38,7 +38,7 @@ final List<ReusableCard> intubation = [
     routeTo: Routes.intubationChecklist,
   ),
   const ReusableCard(
-    title: 'Algorithm',
+    title: 'Infographics',
     description: '12 steps',
     color: AppColors.backgroundGreen,
     routeTo: Routes.intubationAlgorithm,
```

Comment:
You are right. I was thinking if I should do it because the route and variable names should be more general otherwise we'd have to change it all the time when the design is changed.

Reply:
Good point. I guess if it was originally a typo, it should be changed. If it's a design change, then leaving it is ok.